index.html

```diff
@@ -0,0 +1,457 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <h1 id="logo">Aziz Mejri</h1>
+
+      <nav class="nav">
+        <ul class="nav-menu desktop">
+          <li class="nav-item">
+            <a href="#home">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#about">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact">Contact</a>
+          </li>
+          <li class="nav-item desktop">
+            <a href="#mail">
+              <img src="./assets/images/icons/Icon-Mail.svg" alt="mail" />
+            </a>
+          </li>
```

Comment:
♻️ It is a good idea to improve and add new links to your portfolio so you give a better UI/UX. However, as per the requirements, your design should match the one provided. 

- [ ] Please remove this menu item so the menu resembles the actual template:
![image](https://user-images.githubusercontent.com/26911718/204091020-4b6967da-e401-4700-bb4d-d752b012f3d8.png)